src/test/examples/1_HarmonicFuntions/major/sikorski_zzip_ex77.txt

```diff
@@ -1,11 +1,11 @@
 A
 3/4
 D{"position":"3", "system":"close"}
-T{"revolution":"3"};S{"mode":"minor"}
-T{};S{"position":"3"}
+T{"revolution":"3"}; S{"mode":"minor"}
+   T{}    ;      S{"position":"3"}
 D{"position":"5","revolution":"3"}
```

Comment:
Rozumiem, że to na dowód działania. Jednak lepiej byłoby stworzyć osobny test, a w tym nie generować niepotrzebnych spacji